src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -45,9 +45,7 @@ HttpStatusCode mongoNotifyContextAvailability
   const std::string&                   servicePath
 )
 {
-    // FIXME P10: we have been discussing about changing the "notifyFormat" name for something more neutral and more
-    // meaningfull for *both* subscription notification and cpr forwards
-    std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
+    const std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

Comment:
Probably this is a stupid question due to my bad memory :) but... why we need notifyFormat in the mongoNotifyContextAvailability operation?